auth/tokens.go

```diff
@@ -38,6 +40,60 @@ func InjectToken(ctx context.Context, c echo.Context, user *models.Person, sessI
 	return nil
 }
 
+type RLSPayload struct {
+	Tags   []map[string]string `json:"tags,omitempty"`
+	Agents []string            `json:"agents,omitempty"`
+}
+
+func GetAgentsAndTagPermission(ctx context.Context) (*RLSPayload, error) {
+	permissions, err := rbac.PermsForUser(ctx.User().ID.String())
```

Comment:
Can we not use GetImplicitPermissions here?

Reply:
`rbac.PermsForUser` calls `enforcer.GetImplicitPermissionsForUser`